kiota_authentication_azure/_observability.py

```diff
@@ -0,0 +1,33 @@
+from kiota_http.middleware.parameters_name_decoding_handler import ParametersNameDecodingHandler
+from kiota_http.observability_options import ObservabilityOptions
+from opentelemetry import trace
+
+from ._version import VERSION
+
+tracer = trace.get_tracer(ObservabilityOptions.get_tracer_instrumentation_name(), VERSION)
+
+
+class Observability:
+    """Helper object to manage creation of spans."""
+
+    def start_tracing_span(self, uri: str, method: str) -> trace.Span:
```

Comment:
Do we really need the uri? The span name is equivalent to the current method name e.g `get_authorization_token`

Reply:
It's unnecessary. I have updated this now.